doc/classes/XROrigin3D.xml

```diff
@@ -5,20 +5,19 @@
 	</brief_description>
 	<description>
 		This is a special node within the AR/VR system that maps the physical location of the center of our tracking space to the virtual location within our game world.
-		There should be only one of these nodes in your scene and you must have one. All the XRCamera3D, XRController3D and XRAnchor3D nodes should be direct children of this node for spatial tracking to work correctly.
+		Multiple origin points can be added to the scene tree, but only one can used at the same time. All the [XRCamera3D], [XRController3D], and [XRAnchor3D] nodes should be direct children of this node for spatial tracking to work correctly.
```

Comment:
```suggestion
		Multiple origin points can be added to the scene tree, but only one can used at a time. All the [XRCamera3D], [XRController3D], and [XRAnchor3D] nodes should be direct children of this node for spatial tracking to work correctly.
```

Reply:
"at the same time" in the class reference is more common than "at a time", but either way, it's fine.

Reply:
To me this is a singular vs plural thing. Two (or more things) happen "at the same time", but one thing happens "at a time". But I'm from the US so there could be differences in other dialects